src/main/java/racingCar/domain/Cars.java

```diff
@@ -0,0 +1,32 @@
+package racingCar.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Cars {
+    private final List<Car> carList;
```

Comment:
```suggestion
public class Cars {
    private final List<Car> cars;
```
일급컬랙션 잘 추가해주셨네요 👍 

네이밍의 경우 타입이 명시되는 `carList`보다는 `cars`와 같은 복수의 형태를 구성하는 것이 네이밍이 타입에 의존되지 않습니다 😄